Example/Podfile

```diff
@@ -1,6 +0,0 @@
-source 'https://github.com/CocoaPods/Specs.git'
-use_frameworks!
-
-target 'YelpAPI_Example', :exclusive => true do
```

Comment:
That's an unfortunate quirk of the way Xcode handles schemes, it only generates them the first time you open the project. If you go to Product > Scheme > Manage Schemes... and click the "Autocreate Schemes Now" button, then you'll see it.

This will only impact people who had a checkout from before this change. (So, maybe just you and me :P)

Here's what the schemes look like on a fresh checkout for me:
<img width="693" alt="screen shot 2016-04-27 at 11 35 38 am" src="https://cloud.githubusercontent.com/assets/478311/14863935/9f595ac4-0c6c-11e6-83fd-678f131c500a.png">